core/src/autogluon/core/__init__.py

```diff
@@ -12,6 +14,6 @@
 __logging.getLogger("distributed.logging.distributed").setLevel(__logging.ERROR)
 __logging.getLogger("distributed.worker").setLevel(__logging.ERROR)
 
-__logging.basicConfig(format='%(message)s')  # just print message in logs
+_add_stream_handler()
```

Comment:
While we are at it, we can delete lines 8-15 ( the `__logging` lines), since this is very old code and was related to our old usage of dask a long time ago, which has been completely removed.